bot/exts/filters/webhook_remover.py

```diff
@@ -32,7 +32,7 @@ def mod_log(self) -> ModLog:
         """Get current instance of `ModLog`."""
         return self.bot.get_cog("ModLog")
 
-    async def delete_and_respond(self, msg: Message, redacted_url: str) -> None:
+    async def delete_and_respond(self, msg: Message, redacted_url: str, webhook_deleted: bool) -> None:
```

Comment:
Making `webhook_deleted` a kwarg only param would make the interface clearer from the caller